src/constitution.md

```diff
@@ -0,0 +1,191 @@
+
+Each committee is governed by a corresponding Committee Policy Document, maintained separately from the Constitution.
+
+## Appointments and Elections
+
+Members of the Club Executive are by appointment only, except when the current executive cannot agree on qualified candidates or when there is no qualified candidate for a role.
+If an Executive Role cannot be filled by appointment, an announcement will be made at a general meeting advertising the open positions. Candidates may apply by email, at the discretion of the current Club Executive.
+If there are more qualified candidates than open roles, the current Executive may vote --- with a quorum of 50% of filled Executive roles --- to decide which candidate(s) will fulfill the open role(s).
+
+# Meetings
+
+ i. General meetings
+
+    - Must be held regularly (at least bi-weekly) through each semester of operation
+    - Are operated in accordance to the GCC General Meeting Policy
+
+ii. Executive meetings
+
+    - Must occur at least once per semester; other meetings may be held as needed
+    - Are operated in accordance to the GCC Executive Meeting Policy
+
+# Financial Procedures
+
+  i. A budget will be presented to the Executive at the beginning of each semester, and must be approved by majority vote with a quorum of 50% of filled Executive roles.
+ ii. Once internally approved, Club Budgets will be drafted, reviewed, and submitted to SOCIS at the beginning of each semester in accordance to the SOCSSO Policy.
+iii. Receipts of all purchases, bank statements, and/or transactions through each semester must be kept on file in case of audit by the SOCIS Vice President of Finances.
+
+
+# Definitions
+
+CEPSSC
+
+: College of Engineering and Physical Sciences Student Council.
+  The parent organization of SOCIS.
+
+SOCIS
+
+: Society of Computing and Information Science.
+  The parent organization of the Guelph Coding Community.
+
+SOCSSO
+
+: School of Computer Science Student Organization.
+  Operates within the terms mandated by the SOCIS SOCCSO Policy.
+
+PDR
+
+: Petitions, Delegations, and Representations.
+  A process by which additional funding may be obtained by registered student organizations of CEPSSC and SOCIS.
+
+GitHub Organization
+
+: The organization of code repositories found at <https://github.com/guelphcodingcommunity> within which this document may be found.
+
+Committee Policy Document
+
+: A brief document outlining the purpose and operations of a committee within the Guelph Coding Community.
+
+
+# Constitutional Review
+
+The Constitution (this document) shall be reviewed by the President in advance of any time the club needs to be accredited.
+All proposed changes to the club Constitution shall be brought before the Club Executive for review, with a vote to pass by a majority vote.
```

Comment:
Is there a quorum?

Reply:
Should be  :)

Reply:
Set quorum to 2/3 of filled executive body, and clarified "simple majority".